store-and-retrieve-data/index.md

```diff
@@ -5,55 +5,57 @@ description: Learn about the complete data storage and retrieval process in Data
 
-- Create a bucket and choose an MSP. Access is currently owner-only; sharing/public access is on the roadmap. For confidentiality, you can encrypt files on the client side before uploading.
-- Upload files: the app creates a storage request; once required BSP replication acks arrive, the MSP anchors a per-bucket update on-chain that commits to your file(s).
-- Retrieve files later with a lightweight integrity check against the bucket’s on-chain commitment (only when fulfilled).
+- **Bucket access**: Today, access is owner-only. Sharing and public access are planned.
+- **Data security**: For confidentiality, you can encrypt files on the client side before uploading.
+- **Costs and deposits**: Storage is billed via a prepaid deposit that streams per block and auto-pauses when the balance falls below a minimum threshold.
+- **MSP choice**: If you opt to reassign a bucket to a new MSP later, the new MSP rehydrates data from BSP replicas without requiring manual copying.
+- **File status & replicas**: Treat files as “verified” only after a fulfillment signal (e.g., `StorageRequestFulfilled`).
+- **Large files**: Files up to 2 GB are currently supported, subject to change. The app automatically handles chunking and integrity checks under the hood.
 
-Before you jump into guides, here are a few practical notes.
+## Use StorageHub SDK
 
-## Key Considerations
+Use the following how-to guides to move through the DataHaven storage and retrieval lifecycle:
 
-As you plan your workflow, consider the following concepts:
+[timeline.left(datahaven-docs/.snippets/store-and-retrieve-data/overview/timeline-01.json)]
 
-- **Costs and deposits**: Network fees apply. Storage is billed via a prepaid deposit that streams per block and auto-pauses when the balance falls below a minimum threshold.
-- **MSP choice and capacity**: You pick a MSP. Market dynamics encourage MSPs to offer capacity and performance at competitive prices. You can reassign a bucket to a new MSP later; the new MSP rehydrates from BSP replicas (no manual copying required).
-- **File status & replicas**: Treat files as “verified” only after a fulfillment signal (e.g., `StorageRequestFulfilled`) and, ideally, show current replicas vs target so users see progress.
-- **Large files**: Files up to 2 GB are currently supported (subject to change). The app automatically handles chunking and integrity checks under the hood.
+You can also visit the [End-to-End StorageHub SDK Workflow](/store-and-retrieve-data/use-storagehub-sdk/end-to-end-storage-workflow.md) tutorial for a streamlined walk-through using DataHaven's StorageHub SDK to store and retrieve data.
 
 ## Next Steps
 
 <div class="grid cards" markdown>
 
--   **Quick Start**
+-   **Quickstart**
 
     ---
 
-    Create a bucket and upload your first file.
+    Visit the Store and Retrieve Data Quickstart to set up your RPC endpoints, discover block explorers, and obtain TestNet HAVE tokens. 
```

Comment:
We're in the Store and Retrieve data section, so I think that's unnecessary. Would like to keep these descriptions concise. Also, same comment here re: Quick Start vs Quickstart